docs/changelogs.rst

```diff
@@ -106,3 +106,13 @@ The full list of categories is:
 
 **trivial**
   This category will **not be shown** in the changelog. It can be used to describe changes that are not touching user-facing code, like changes in tests. This is useful if every PR is required to have a changelog fragment.
+
+Porting Guide Entries
+======================
+
+The changelog generator considers the following categories as part of a Porting Guide section:
```

Comment:
The changelog generator doesn't really know about porting guides. How about:
```suggestion
Ansible includes the following categories into its Porting Guide:
```
or
```suggestion
The following sections are considered as the Porting Guide of the collection. For collections included in Ansible, these will be inserted into Ansible's Porting Guide:
```
I can add a feature which allows to export these sections as a separate `porting_guide.rst` if people want this.